### PR TITLE
remove support for deploying as an aws lambda application

### PR DESCRIPTION
**Pull request by asjohnston-asf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,9 +21,6 @@ These options allow you to specify an AWS S3 bucket path where the browse image
 2. Installing `gdal` via pip requires that `git`, `gcc`, and `g++` are installed on your system.
```
Comment (forrestfwilliams) [inline README.md:21]: Bit of a nit-pick, but this issue isn't something a user needs to worry about if they're installing via Conda, which is what we recommend. Feel free to change if you agree.

File: requirements.txt
Comment (forrestfwilliams) [inline requirements.txt:1]: Do we need this file at all if we're recommending people install this package via Conda? I imagine we'll end using Conda in the docker container as well.

**Review by forrestfwilliams — APPROVED** [review]
Left two nit-pick comments, but I wouldn't hold of merging to address these. Especially since I won't be around this afternoon to give it a re-approval.

File: requirements.txt
Comment (asjohnston-asf) [reply requirements.txt:1]: Yes, I want to revisit package management as we transition from a lambda application to a docker application. I'd designed the pyproject.toml, requirements.txt, requirements-develop.txt, and environment.yml to try to provide the best of:
1. packaging source code as an installable module
1. dependabot support
1. support for both conda-installed development environments and pip-installed lambda environments
1. not including development requirements in production deployments

That might be overkill if we're now intending this code to only ever be run as a docker application.

**Pull request merged**